.prettierignore

```diff
@@ -4,3 +4,5 @@ yarn.lock
 .prettierignore
 .gitattributes
 site
+
+src/server/utils/tcp-packet.ts

```

Comment:
Custom formatting is not my favorite, especially since e.g. editors like to autoformat on save so it usually gets wrecked anyways.

Reply:
well- i want to keep the spacing on the packet variables such that the arguments are easy to identify. prettier doesnt like it, but the editor is fine with it